tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compileOnSave": true,
+
+  "compilerOptions": {
+    "module": "commonjs",
+    "allowUnreachableCode": false,
+    "alwaysStrict": true,
+    "strictNullChecks": true,
+    "allowSyntheticDefaultImports": true,
+    "noImplicitAny": true,
```

Comment:
I'd add noImplicitReturns to true, declarations to true, noUnusedLocals: true that's all for tsconfig for now, if you have something in mind which will improve code quality better then please add, there are so many option, those are the ones I use.